governance/governance.go

```diff
@@ -176,6 +181,10 @@ func (w *GovernanceWorker) NewEvent(incoming events.Message) {
 			cmd := producer.NewExchangeMessageCommand(*msg)
 			w.Commands <- cmd
 		}
+
```

Comment:
This is the exchange message event, why do you generate the report just because there is a message from an agbot?

Reply:
Good catch. It meant to put where the device id and token were assigned.  I have deleted it.